app/lib/presenter/complaint.rb

```diff
@@ -15,6 +15,11 @@ def optics_payload
 
     private
 
+    def request_date
+      time = @data.fetch(:submissionDate, Time.now.nsec.to_s)
+      Time.strptime((time.to_i).to_s, '%N').iso8601.split('T')[0]
```

Comment:
`.split('T')[0]` seems a bit unnecessary seeing as we have Rails niceness. 

Nicer way could be:
```
Time.at(time/1000).to_date
```

This also works with the Optics API